ui/src/message_popup/author/question_page.jsx

```diff
@@ -104,8 +106,11 @@ export default React.createClass({
     console.log("Saving the following new question:");
     const studentIds = this.state.students.map(student => student.id);
     const text = this.state.questionText;
+    const {originalQuestion} = this.props;
     const {goodExamples, badExamples} = this.getExamples();
-    const id = _.maxBy(allQuestions, question => question.id).id + 1;
```

Comment:
👍 Let's do it!  And to clarify a bit I think there's two separate issues: one is the "how do I load questions from the server and then pass them to a child component once they're loaded" and the other is "when an update/create/delete is made to the set of questions, how do I send those back to the server and update the list of questions in the browser."